test/dune.inc

```diff
@@ -171,4 +171,11 @@
  (action (progn
            (run ocaml-mdx test --direction=infer-timestamp %{x})
 
+           (diff? %{x} %{x}.corrected))))
+(alias
+ (name   runtest)
+ (deps   (:x section.md))
+ (action (progn
+           (run ocaml-mdx test --direction=infer-timestamp -s "Testing" %{x})
+
```

Comment:
We really need to fix that empty line bug!

Reply:
Fixed on master.